SECURITY.md

```diff
@@ -0,0 +1,98 @@
+# Security Policy
+
+Pyogrio wraps [GDAL](https://gdal.org/), which depends on a variety of third-
+party dependencies, such as [GEOS](https://libgeos.org/),
+[PROJ](https://proj.org/), and [CURL](https://curl.se/). These dependencies
+vary based on which features are enabled when GDAL is compiled and vary across
+platforms.
+
+Most vulnerabilities are likely to occur in these dependencies.
+
+Please see [GDAL's security policy and published advisories](https://github.com/OSGeo/gdal/security).
+
+If you know that a vulnerability originates from a third-party dependency,
+please report the vulnerability directly to the affected dependency.
+
+If the vulnerability requires a modification of Pyogrio's packaging to
+specifically avoid linking to a known vulnerable version of a third party
+dependency, please report the vulnerability here as well.
+
+## Available packages
+
+Pyogrio is available in 3 basic forms:
+
+### Binary wheels on PyPI
+
+Binary wheels are published to [PyPI](https://pypi.org/project/pyogrio/) and
+are created using [VCPKG](https://vcpkg.io/en/) versions of GDAL and associated
+dependencies. These wheels include binary libraries of these dependencies.
+Because these binary wheels are specifically under our control, these are the
+packages where we are most concerned with ensuring that we avoid known
+vulnerable versions of dependencies.
+
+### Conda-forge packages
+
+Conda packages are available on
+[conda-forge](https://anaconda.org/conda-forge/pyogrio). Pyogrio uses packages
+for GDAL and associated dependencies. Please contact the appropriate maintainers
+for those packages to report vulnerabilities.
+
+### Self-compiled / local development
+
+When you build Pyogrio locally, you link it to your local version of GDAL and
+associated dependencies in a way that depends on how you installed GDAL and its
+dependencies (e.g., system package, homebrew package, etc). Please contact the
+appropriate maintainers of these packages to report vulnerabilities.
+
+## Supported versions
+
+Pyogrio has not yet reached 1.0. Only the latest available version is being
+supported with security updates.
```

Comment:
Will this ever be different? I.e. do you assume we will support older major versions by patching them?

Reply:
Once we hit 1.0, we _may_ consider backporting fixes to a previous (but still relatively recent) version.